Description: corrected help target
 help target is based on the python script docstring, which is
 incorrectly positioned, making python falling back to "None".
 This patch correct the position.
Author: Philippe Thierry
Last-Update: 2017-06-16
--- a/pig.py
+++ b/pig.py
@@ -1,9 +1,5 @@
 #! /usr/bin/env python
 # -*- coding: utf-8 -*-
-from scapy.all import *
-import string, binascii, signal, sys, threading, socket, struct, getopt
-from sys import stdout
-
 """
 enhanced DHCP exhaustion attack.
 
@@ -47,6 +43,11 @@
     -c, --color                    ... enable color output (off)
 """
 
+from scapy.all import *
+import string, binascii, signal, sys, threading, socket, struct, getopt
+from sys import stdout
+
+
 
 class Colors:
     class Palette:
